README.md

```diff
@@ -0,0 +1,100 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# To-do-List
+   In this project, build a simple list of To Do tasks using webpack and served by a webpack dev server.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+- Webpack
+
+## Key Features
+- [Setting up of Linters]
+- [Setting up of webpack]
+- [Adding Index.html and style.css]
+- [styling a text using class giving a appropriate font size,margin,image,padding]
+
+## Live Demo
+
+[Live Demo]()
```

Comment:
- [ ] Since you have not yet deploy the App, It would be nice to remove the empty link under the **Live Demo** for now. You can put a simple text showing that the live demo is not ready yet (**e.g.** `coming soon` or `not available yet` ...) or simply remove the session. That will make you README file look more professional.